src/rantanplan/core.py

```diff
@@ -682,20 +709,26 @@ def _get_scansion(text, rhyme_analysis=False, rhythm_format="pattern",
             structure_length = rhythmical_lengths
         else:
             line_structure = line.get("structure", None)
-            structure_length = STRUCTURES_LENGTH.get(line_structure, None)
+            structure_length, cyclic_structure = STRUCTURES_LENGTH.get(
+                line_structure, [None, None])
+            if structure_length is not None and cyclic_structure:
```

Comment:
Im not evaluating the `[None, None]` just the first element. Also, can't be just `[]` beacuse I'm assigning two variables and it will fail, it has to be a list. `[[], False]` makes thests fail and I whould investigate a bit and not sure if it's worth it, what about `[None, False]`?

Reply:
Forget it, just did waht you said :)